updates/16/updates.sql

```diff
@@ -301,4 +301,6 @@ INSERT INTO assets(identifier, name, type) VALUES("IP3", "Cripco", "Y"); INSERT
 INSERT INTO assets(identifier, name, type) VALUES("eip155:1/erc20:0xB755D5bC7de83232b9df1886bD5CdB38895933B0", "hiMFERS", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:1/erc20:0xB755D5bC7de83232b9df1886bD5CdB38895933B0", "A", 1, "0xB755D5bC7de83232b9df1886bD5CdB38895933B0", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:1/erc20:0xB755D5bC7de83232b9df1886bD5CdB38895933B0", "hiMFERS", "himfers", "", NULL, 1672726895, NULL);
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:56/erc20:0xD0ACCF05878caFe24ff8b3F82F194C62Ed755707", "SIMP Token", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:56/erc20:0xD0ACCF05878caFe24ff8b3F82F194C62Ed755707", "A", 56, "0xD0ACCF05878caFe24ff8b3F82F194C62Ed755707", 6, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:56/erc20:0xD0ACCF05878caFe24ff8b3F82F194C62Ed755707", "SIMP", "simp-token", "", NULL, 1633886436, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES("eip155:10/erc20:0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000", "Ether", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:10/erc20:0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000", "A", 10, "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000", 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:10/erc20:0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000", "ETH", "ETH", "ETH", NULL, 1438269973, NULL);
```

Comment:
coingecko id for ethereum is `ethereum` https://www.coingecko.com/en/coins/ethereum.

Also the date seems wrong

![image](https://user-images.githubusercontent.com/5068010/213438885-182e9d12-c997-4f02-97e5-a7d949550ba2.png)

Reply:
Ah yes, I put the date when Ethereum was launched, which is obviously wrong. Thanks